app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/inputDate.scala.html

```diff
@@ -22,16 +22,16 @@
     form(id).errors ++ form(s"$id.day").errors ++ form(s"$id.month").errors ++ form(s"$id.year").errors
 }
 @errorContent(error: FormError) = {
-    <span id="@id-error" class="govuk-error-message govuk-!-margin-bottom-1">
+    <p id="@id-error" class="govuk-error-message govuk-!-margin-bottom-1">
```

Comment:
I'm not sure you need this extra html around your errors, the date input should handle rendering the errors you provide it for you